student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -203,4 +213,35 @@ public void approveAccommodations(UUID examId, ApproveAccommodationsRequest appr
     
     return Optional.absent();
   }
+  
+  @Override
+  public Response<ExamConfiguration> startExam(UUID examId) throws ReturnStatusException {
+    HttpHeaders headers = new HttpHeaders();
+    headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+    headers.setContentType(MediaType.APPLICATION_JSON);
+    HttpEntity<?> requestHttpEntity = new HttpEntity<>(headers);
+    Response<ExamConfiguration> response;
+    UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(String.format("%s/%s/start", examUrl, examId));
+  
+    try {
+      ResponseEntity<Response<ExamConfiguration>> responseEntity = restTemplate.exchange(
+        builder.build().toUri(),
+        HttpMethod.PUT,
+        requestHttpEntity,
+        new ParameterizedTypeReference<Response<ExamConfiguration>>() {
+        });
```

Comment:
I agree it is a good idea.  However, I don't think implementing that needs to be part of this PR.